### ISSUE TITLE
Weights set to zero and reps to beat overflowed(?)

### ISSUE THREAD

**Issue by EricKirschenmann** [jnordstrom90/Wendler#13]
So something weird happened to my week 4, all the sets are 0 and the reps to beat looks like it overflowed to the negative end of a signed 32-bit integer. All my other weeks and all the other lifts look alright and I didn't do anything that I know of to mess this up so I don't know.

![screenshot_2015-02-27-11-30-10](https://cloud.githubusercontent.com/assets/7422193/6420664/f5c95cae-be7c-11e4-8e66-12b93a9e3015.png)

**Comment by jnordstrom90** [comment]
Why would you set your weights to 0?

**Comment by PehrGit** [comment]
I don't think he meant to set his weights to 0

**Comment by EricKirschenmann** [comment]
That's the problem, it lost the weights or something. I have 390 as my week 3 on deadlift and then this week 4.

**Comment by jnordstrom90** [comment]
Ok. I'll have a look at it when I get the time, but currently this project is dormant for me. Feel free to create a PR for fix.

**Comment by EricKirschenmann** [comment]
Yeah I think it might have something to do with backing up and restoring. Luckily I was done with the cycle so I just cleared the data and started over and it's fine now.
